src/app/appservice.service.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AppserviceService {
+
+  constructor(private http:HttpClient) {
+
+
+   }
+
+
+   login(user:any){
+
+
+    let url="http://localhost:8000/users/getUserByEmail/"+user.email
```

Comment:
instead of creating urls in the method it can be declared outside..

Reply:
Completed